sky/cli.py

```diff
@@ -1328,14 +1328,24 @@ def queue(clusters: Tuple[str], skip_finished: bool, all_users: bool):
     default=False,
     help=('If specified, do not show logs but exit with a status code for the '
           'job\'s status: 0 for succeeded, or 1 for all other statuses.'))
+@click.option('--follow/--no-follow',
+              is_flag=True,
+              default=True,
+              help='Follow the logs of the job. [default: --follow]')
```

Comment:
Maybe add: `--no-follow to print the log so far and exit.`.

Reply:
Added. Thanks!